### DISCUSSION TITLE
Container losing network connections

### DISCUSSION THREAD

**Discussion by github-cli** [louislam/dockge#401] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

I use macvlan networks which are all defined in the compose file but when pulling updates or stop/start via dockge, the containers lose their network connection, i need to go to the server cli and docker-compose down / up for the container to work again

### 👟 Reproduction steps

update container

### 👀 Expected behavior

container still works

### 😓 Actual Behavior

container is not reachable anymore

### Dockge Version

1.4.2

### 💻 Operating System and Arch

20.04

### 🌐 Browser

Chrome 120.0

### 🐋 Docker Version

24.0.5

### 🟩 NodeJS Version

?

**Comment by louislam** [comment]
Dockge is basically calling `docker-compose up/down` in the back scene. It don't touch any docker logic in deep. You should check your network instead.
